src/network/http/websocket.cpp

```diff
@@ -188,7 +165,30 @@ namespace fc { namespace http {
               return _ws_connection->get_request_header(key);
             }
 
+            /****
+             * @brief retrieves the remote hostname
```

Comment:
IP not hostname :)

Reply:
changed at https://github.com/bitshares/bitshares-fc/pull/134/commits/7ec8ead573ea087dd16e0e3bfbcc8afe027e972d